### PR TITLE
[TEVA-1100] Refactor new sections notification

### PR DESCRIPTION
**Pull request by cpjmcquillan**
This PR refactors the new sections notification.

## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1100

## Changes in this PR
- Use notification component to render action required and new sections on review/edit pages
- Add new section message for contact number

### CODE REVIEW

File: app/frontend/styles/components/flashes.scss [hunk 1]
```diff
@@ -40,6 +40,10 @@
 
 .govuk-notification--notice {
   border-color: govuk-colour('blue');
+
+  .govuk-notification__list a {
+    color: govuk-colour('blue');
```
Comment (david-mears-dfe) [inline app/frontend/styles/components/flashes.scss:45]: Isn't blue the default for <a>? I'd be tempted to just specify when you need it to not be the default color

File: app/views/hiring_staff/vacancies/_new_tag.html.haml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- attributes.each do |attribute|
```
Comment (david-mears-dfe) [inline app/views/hiring_staff/vacancies/_new_tag.html.haml:1]: The filename isn't obvious enough for my liking. Can we do e.g. _new_attributes_tag?

**Review by david-mears-dfe — APPROVED** [review]
LG to M

File: app/views/hiring_staff/vacancies/_new_tag.html.haml (hunk 2 above)
Comment (cpjmcquillan) [reply app/views/hiring_staff/vacancies/_new_tag.html.haml:1]: Certainly

File: app/frontend/styles/components/flashes.scss (hunk 1 above)
Comment (cpjmcquillan) [reply app/frontend/styles/components/flashes.scss:45]: Indeed it is, copy pasta got out of hand 🍝

**Pull request merged**